### PR TITLE
feat(functions/v2): use FF testing layer for GCS sample

### PR DESCRIPTION
**Pull request by saraford**

### CODE REVIEW

File: functions/v2/helloGCS/.gcloudignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules/
```
Comment (ace-n) [inline functions/v2/helloGCS/.gcloudignore:1]: Can we drop this file in favor of one in `functions/v2`?

**Review by ace-n — APPROVED** [review]

File: functions/v2/helloGCS/test/unit.test.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      type: 'mock-gcs-event',
+      data: {
+        bucket: 'my-bucket',
+        name: 'my-file.txt',
+      },
+    };
+
+    // Call tested function and verify its behavior
+    const helloGCS = getFunction('helloGCS');
+    helloGCS(event, context);
```
Comment (ace-n) [inline functions/v2/helloGCS/test/unit.test.js:47]:
```suggestion
    helloGCS(event);

File: functions/v2/helloGCS/.gcloudignore (hunk 1 above)
Comment (ace-n) [reply functions/v2/helloGCS/.gcloudignore:1]: @saraford and I decided (offline) not to do this.

**Review by ace-n — APPROVED** [review]
Confirmed tests pass locally. 👍 

@saraford thanks for helping out on this! 🙂

**Review by sofisl — APPROVED** [review]

**Pull request merged**